src/Dropdown/index.jsx

```diff
@@ -95,10 +115,22 @@ DropdownToggle.defaultProps = {
   bsPrefix: 'dropdown-toggle',
 };
 
+Dropdown.Item = ({ className, ...otherProps }) => (
+  <DropdownItem
+    className={classNames(className, 'pgn__dropdown-item')}
+    {...otherProps}
+  />
+);
+Dropdown.Item.propTypes = {
+  className: PropTypes.string,
+};
+Dropdown.Item.defaultProps = {
+  className: '',
```

Comment:
nit: `undefined` preferred (using `''` for a class name can result in an unnecessary/empty class attribute in the DOM).

Reply:
fair